games/devtest/mods/unittests/misc.lua

```diff
@@ -68,3 +68,25 @@ local function test_clear_meta(_, pos)
 	end
 end
 unittests.register("test_clear_meta", test_clear_meta, {map=true})
+
+local function test_compress()
+	-- This text should be compressible, to make sure the results are... normal
+	local text = "The\000 icey canoe couldn't move very well on the\128 lake. The\000 ice was too stiff and the icey canoe's paddles simply wouldn't punch through."
+	local methods = {"deflate"}
+	if not core.features.compress_zstd then
+		core.log("warning", "test_compress: Server too old, skipping zstd test.")
```

Comment:
I don't think you need this check. There's no reason anyone would use a newer devtest with an older Minetest. It's probably not supported anyways (since devtest relies on newer features).

Reply:
98fad9fb3 now removes this check.